protobuf-test-suite/codegen.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+cd `dirname $0`
+PROTO_PATH="proto"
+PROTO_PREFIX="HSCodeGen"
+HASKELL_SRC_PATH="src/"
+
+stack exec hprotoc -- \
+  --haskell_out=$HASKELL_SRC_PATH \
+  --proto_path=$PROTO_PATH \
+  --prefix=$PROTO_PREFIX \
+  --unknown_fields `find $PROTO_PATH  -type f | egrep '\.proto$'`
+
+cd "$HASKELL_SRC_PATH"
+EXPOSED_MODULES=`find * -print | egrep '\.hs$' | egrep "$PROTO_PREFIX" | cut -f 1 -d '.' | tr / .`
+cd -
+
+display_modules () {
+  for i in ${1}; do
+    printf "\n%"$2"s$i";
+  done
+}
+
+CABAL_FILE="protobuf-test-suite.cabal"
+
+OTHER_MODULES=""
+[ -f "$CABAL_FILE" ] && OTHER_MODULES=`egrep 'other-modules' "$CABAL_FILE" -A10000`
```

Comment:
Do we need to generate code like this? I think hpack https://github.com/sol/hpack might be a nicer way to do this.